### PR TITLE
Add new Cell Ontology terms and modify several existing terms

### PR DESCRIPTION
**Pull request by 1136514475**

### CODE REVIEW

**Comment by dosumis** [comment]
Edits look reasonable although:
(i) need to be moved to cl-edit.owl and released through normal process.  
(ii) Name change from Clara cell -> club cell needs a reference & retention of old name as synonym. (https://www.ncbi.nlm.nih.gov/pubmed/12107102 support Clara cell).

It's a bit odd to get a pull request from a brand new & anonymous account.  Is this a programatic edit?  And what/who is CellBlast? For traceability and proper credit I'd like to be able to attribute edits.  

Many thanks,
David

**Comment by 1136514475** [comment]
Dear David,

I have moved the edits to cl-edit.owl. 
A reference (https://www.ncbi.nlm.nih.gov/pubmed/28128362) is added for club cell, and Clara cell is kept as synonym. 

Cell BLAST is an accurate and robust cell-querying method. You can see our preprint "Cell BLAST: Searching large-scale scRNA-seq databases via unbiased cell embedding" on bioRxiv.

If there's anything else I can do for you, just let me know.

Many thanks,
Cell BLAST

**Comment by addiehl** [comment]
Clara cell -> club cell name change explained here:
https://journal.chestnet.org/article/S0012-3692(13)60001-X/fulltext
https://erj.ersjournals.com/content/36/4/722.long

tldr:  Clara was a Nazi with a dubious political history and in 2012 CHEST and a number other journals decided to rename Clara cell to club cell.

**Comment by 1136514475** [comment]
Okay, I just modified the references for club cell.

**Comment by matentzn** [comment]
edits to cl-basic.obo and cl.obo should be reverted.

**Comment by matentzn** [comment]
(they should never be edited directly!)

**Comment by 1136514475** [comment]
Edits to cl-basic.obo and cl.obo has already been reverted in the third commit in this pull request.

**Comment by dosumis** [comment]
See https://github.com/obophenotype/cell-ontology/pull/569

**Comment by addiehl** [comment]
What is the status of this PR?  Can these minor changes be implemented or are there still issues with cl-basic.obo and cl.obo?

**Comment by dosumis** [comment]
> What is the status of this PR? Can these minor changes be implemented or are there still issues with cl-basic.obo and cl.obo?

I redid this using patterns.  Changes are in #569.  Will work on merge now.
